Formula/steem.rb

```diff
@@ -0,0 +1,32 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com"
+  url "https://github.com/steemit/steem", :using => :git, :tag => "v0.8.5b", :revision => "0ea6049e760e0452ecb22b98a2ad808493bd3760"
```

Comment:
Also, please split this out a bit more so it doesn't rampage past 80 characters quite so much.
```ruby
url "https://github.com/steemit/steem.git",
    :tag => "v0.8.5b"
    :revision => "0ea6049e760e0452ecb22b98a2ad808493bd3760"
```
etc. Does this need building from a git checkout rather than archive tarball for a specific reason?